spacecmd/src/spacecmd/ssm.py

```diff
@@ -92,9 +92,8 @@ def do_ssm_add(self, args):
         if system in self.ssm:
             logging.warning(_N('%s is already in the list') % system)
             continue
-        else:
-            self.ssm[system] = self.get_system_id(system)
-            logging.debug('Added %s' % system)
+        self.ssm[system] = self.get_system_id(system)
+        logging.debug('Added %s' % system)
```

Comment:
This string can also be constructed lazily (i.e. only if debug messages are actually logged):
```suggestion
        logging.debug('Added %s', system)
```